RecoJets/JetProducers/plugins/VirtualJetProducer.h

```diff
@@ -178,7 +182,12 @@ class dso_hidden VirtualJetProducer : public edm::stream::EDProducer<>
   bool                  doRhoFastjet_;              // calculate rho w/ fastjet?
   bool                  doFastJetNonUniform_;       // choice of eta-dependent PU calculation
   double                voronoiRfact_;              // negative to calculate rho using active area (ghosts); otherwise calculates Voronoi area with this effective scale factor
-  
+
+  double                rhoEtaMax;                  // Eta range of jets to be considered for Rho calculation; Should be at most (jet acceptance - jet radius)
```

Comment:
please continue to follow the coding convention in this class (trailing "_" for member data names)